src/components/startPage/StartPage.js

```diff
@@ -16,9 +18,35 @@ function StartPage() {
         setFields(newFields);
     }
 
+    async function buttonBack() {
+        const newFields = { ...field };
+        if (newFields.page !== 1) newFields.page = newFields.page - 1;
+        setFields(newFields);
+        const response = await axios.get(`https://api.flickr.com/services/rest?sort=relevance&parse_tags=1&content_type=7&extras=description%2Cowner_name%2Crealname%2Curl_s%2Curl_n%2Curl_w%2Curl_m%2Curl_z%2Curl_c%2Curl_l&per_page=${newFields.perPage}&page=${newFields.page}&lang=en-US&text=${newFields.keyWord}&viewerNSID=&method=flickr.photos.search&csrf=&api_key=bfe6ce4d5f9d4e425f7bf0b9db4fa329&format=json&hermes=1&hermesClient=1&reqId=5541ed04&nojsoncallback=1`)
```

Comment:
remove unused params, probably could be:
const response = await axios.get(`https://api.flickr.com/services/rest?parse_tags=1&content_type=7&per_page=${newFields.perPage}&page=${newFields.page}&lang=en-US&text=${newFields.keyWord}&method=flickr.photos.search&csrf=&api_key=bfe6ce4d5f9d4e425f7bf0b9db4fa329&format=json&hermes=1&hermesClient=1&reqId=5541ed04&nojsoncallback=1`)
**api_key** should be passed dyncamically